packages/design-system-scripts/gulp/common/copyFontsImages.js

```diff
@@ -15,10 +15,21 @@ function minimizeSvg(srcGlob, dest) {
           plugins: [
             {
               cleanupIDs: false,
+            },
+            {
               removeTitle: false,
+            },
+            {
               removeDesc: false,
+            },
+            {
               removeHiddenElems: false,
             },
+            {
+              removeUnknownsAndDefaults: {
+                keepRoleAttr: true,
+              },
```

Comment:
@nichia did this work?

Reply:
Yes it worked. I checked the svg file created under `dist/image` to confirm that the `role="image"` option is present